core/chains/evm/config/config_test.go

```diff
@@ -345,16 +294,9 @@ func TestChainScopedConfig_Profiles(t *testing.T) {
 		t.Run(tt.name, func(t *testing.T) {
 			t.Parallel()
 
-			gcfg := configtest.NewGeneralConfig(t, func(c *chainlink.Config, secrets *chainlink.Secrets) {
-				id := ubig.NewI(tt.chainID)
-				cfg := toml.Defaults(id)
-				c.EVM[0] = &toml.EVMConfig{
-					ChainID: id,
-					Enabled: ptr(true),
-					Chain:   cfg,
-				}
+			config := testutils.NewTestChainScopedConfig(t, func(c *toml.EVMConfig) {
+				c.ChainID = ubig.NewI(tt.chainID)
```

Comment:
Why remove the chain ID based defaults from all of these tests?

Reply:
Not sure what you mean, chainID is different for every test case. This will set the chain-specific defaults depending on the chain ID.

Reply:
You removed the line that was doing that: `cfg := toml.Defaults(id)`
Was it redundant? Or has it been moved/replaced by something else?

Reply:
`NewTestChainScopedConfig` underneath `calls toml.Defaults(chainID)` . Take a look [here](https://github.com/smartcontractkit/chainlink/blob/develop/core/chains/evm/testutils/config.go#L17)

Reply:
That uses the fixture ID. The override function is called afterwards

Reply:
Ah nvm, I guess it calls twice